iron-icon.html

```diff
@@ -107,108 +107,112 @@
   </template>
 
   <script>
-
-    Polymer({
-
-      is: 'iron-icon',
-
-      properties: {
-
-        /**
-         * The name of the icon to use. The name should be of the form:
-         * `iconset_name:icon_name`.
-         */
-        icon: {
-          type: String
-        },
-
-        /**
-         * The name of the theme to used, if one is specified by the
-         * iconset.
-         */
-        theme: {
-          type: String
-        },
-
-        /**
-         * If using iron-icon without an iconset, you can set the src to be
-         * the URL of an individual icon image file. Note that this will take
-         * precedence over a given icon attribute.
-         */
-        src: {
-          type: String
-        },
-
-        /**
-         * @type {!Polymer.IronMeta}
-         */
-        _meta: {
-          value: Polymer.Base.create('iron-meta', {type: 'iconset'})
-        }
-
-      },
-
-      observers: [
-        '_updateIcon(_meta, isAttached)',
-        '_updateIcon(theme, isAttached)',
-        '_srcChanged(src, isAttached)',
-        '_iconChanged(icon, isAttached)'
-      ],
-
-      _DEFAULT_ICONSET: 'icons',
-
-      _iconChanged: function(icon) {
-        var parts = (icon || '').split(':');
-        this._iconName = parts.pop();
-        this._iconsetName = parts.pop() || this._DEFAULT_ICONSET;
-        this._updateIcon();
-      },
-
-      _srcChanged: function(src) {
-        this._updateIcon();
-      },
-
-      _usesIconset: function() {
-        return this.icon || !this.src;
-      },
-
-      /** @suppress {visibility} */
-      _updateIcon: function() {
-        if (this._usesIconset()) {
-          if (this._img && this._img.parentNode) {
-            Polymer.dom(this.root).removeChild(this._img);
-          }
-          if (this._iconName === "") {
-            if (this._iconset) {
-              this._iconset.removeIcon(this);
+    (function (window) {
+      window.customElements.whenDefined('iron-meta').then(function () {
```

Comment:
Given the HTML import of iron-meta above, it should _always_ be defined by the time iron-icon is. There's some other bug here. Are we seeing this fix things only with the polyfill or when bundled? cc @usergenic @valdrinkoshi

Reply:
@nikitindv Firefox and browsers which require the custom elements polyfill are vulnerable to having the polyfills broken when extensions (such as LastPass) are installed which patch their own version of the "createElement" method onto "document".  There is a workaround for this particular scenario; can you provide reproduction instructions for the error?